configs/components/facter.rb

```diff
@@ -32,7 +32,7 @@
   if platform.is_osx?
     pkg.build_requires "cmake"
     pkg.build_requires "boost"
-    pkg.build_requires "yaml-cpp --with-static-lib"
```

Comment:
the formula in our private tap defaults to build with static libs and the bottle on our internal mirror is built from that formula.